media-libs/vidstab/vidstab-9999.ebuild

```diff
@@ -0,0 +1,20 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit git-r3 cmake-utils
```

Comment:
the same about cmake

Reply:
I guess you meant the DEPEND? If so - I fixed it.